docs/getting-started/first-abn.md

```diff
@@ -55,44 +55,24 @@ A sample application using the Iter8 SDK is provided. Deploy both the frontend a
 
 ## Describe the application
 
-In order to support `Lookup()`, Iter8 needs to know what the application component versions look like. A `ConfigMap` is used to describe the make up of possible versions:
+In order to support `Lookup()`, Iter8 needs to know what the application component versions look like. A _routemap_ is created to do this. A routemap contains a description of each version of an application and may contain a routing template. To create the routemap:
 
 ```shell
-cat <<EOF | kubectl apply -f -
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: backend
-  labels:
-    app.kubernetes.io/managed-by: iter8
-    iter8.tools/kind: routemap
-    iter8.tools/version: "v0.18"
-immutable: true
-data:
-  strSpec: |
-    versions:
-    - resources:
-      - gvrShort: svc
-        name: backend
-        namespace: default
-      - gvrShort: deploy
-        name: backend
-        namespace: default
-    - resources:
-      - gvrShort: svc
-        name: backend-candidate-1
-        namespace: default
-      - gvrShort: deploy
-        name: backend-candidate-1
-        namespace: default
+cat <<EOF | helm template routing --repo https://iter8-tools.github.io/iter8 routing-actions --version 0.18 -f - | kubectl apply -f -
```

Comment:
Why do we to `helm template` into `kubectl apply`? Is it not possible to do `helm install`?

Reply:
It is not reliable. helm is very particular about modifying only resources it has created. I tried and failed for this reason.